src/commits/commit/tests/mod.rs

```diff
@@ -162,3 +164,116 @@ fn test_non_conventional_commits_fail_linting(commit_message: &str) {
 }
 
 mod generated_tests;
+
+#[rstest(
+    commit_message,
+    case("fix: This is a very long commit message that exceeds the seventy-two character limit"),
```

Comment:
Delete this unit test but move one example into the existing failing_linting unit test.